doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -217,6 +222,8 @@ create_view ::=
   'CREATE' ('TEMP' | 'TEMPORARY')? 'VIEW' view_name 'AS' select_stmt |
   'CREATE' ('TEMP' | 'TEMPORARY')? 'VIEW' 'IF NOT EXISTS' view_name 'AS' select_stmt |
   'CREATE' 'OR REPLACE' 'VIEW' view_name 'AS' select_stmt
+create_views ::=
+  'CREATE' ('TEMP' | 'TEMPORARY')? 'VIEWS' 'IF NOT EXISTS'? 'FROM' 'SOURCE' '"'src_name'"' ( '(' '"' upstream_table '"'  (AS '"' new_view_name '"' )? ')' )?
```

Comment:
missing `MATERIALIZED` modifier and also having the `'(' '"' upstream_table '"'  (AS '"' new_view_name '"' )? ')'` part as a comma separated group